Update interfaces/dom.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Note: This file was recently manually updated in commit 91a5f1ae0c.
      This commit may revert some of those changes.

Source: https://github.com/tidoust/reffy-reports/blob/a1fde73/whatwg/idl/dom.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/579102609

foolip commented:
wpt-firefox-nightly-stability failed on Taskcluster because of two many affected tests, it was in run 9/10 when "Task timeout after 7200 seconds. Force killing container." Admin merging.

foolip commented:
Oops, nope! This is reverting https://github.com/web-platform-tests/wpt/commit/91a5f1ae0c9501100ce9cd476ba8c08a1bff5125, should wait for the changes to propagate to reffy-reports instead.

autofoolip committed:
Update interfaces/dom.idl

Note: This file was recently manually updated in commit 91a5f1ae0c.
      This commit may revert some of those changes.

Source: https://github.com/tidoust/reffy-reports/blob/a1fde73/whatwg/idl/dom.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/579102609

foolip commented:
This adds XPath interfaces, so the tentative idlharness test should be deleted.

foolip commented:
This is part of https://github.com/web-platform-tests/wpt/pull/18790, should be closed when that's merged.
